middleware/nocache.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // Unix epoch time
-var epoch = time.Unix(0, 0).Format(time.RFC1123)
+var epoch = time.Unix(0, 0).Format(http.TimeFormat)
```

Comment:
https://pkg.go.dev/net/http#TimeFormat
"The time being formatted must be in UTC for Format to generate the correct format."

This is not correct

Reply:
sorry my bad, fixed in recent commit